setup/install/providers/aws/aws-ec2.md

```diff
@@ -85,55 +93,234 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
 --parameter-overrides AuthArn=FROM_ABOVE ManagingAccountId=FROM_ABOVE --capabilities CAPABILITY_NAMED_IAM --region us-west-2
 ```
 
-## Option-3 : Use AWS Console UI (Manual Steps) 
+## Option-3 : Configure with AWS IAM Console
+The Spinnaker **Managing** role AWS account Assumes the Spinnaker **Managed** role AWS account in a target AWS account via AWS IAM resources (policies, roles, users, trust relationship, etc). This allows Spinnaker to take control of the AWS cloud resources.
+
+For the Example below the AWS Account **spinnakerManaging** assumes the **spinnakerManaged** role in the AWS accounts **develop** and **staging**. The account **spinnakerManaging** is where Spinnaker lives
+
+A great use case for this set up is to deploy pre-built AWS AMIs to AWS EC2.
+
+![Example AWS IAM structure for Spinnaker AWS Provider](/setup/install/providers/images/example-aws-provider.svg)
 
+Before you start create a simple table mapping the Account names and account IDs for your desired set up. An example table is shown below.
+
+| Name              | Account Id   |
+|-------------------|--------------|
+| spinnakerManaging | 100000000001 |
+| develop           | 200000000002 |
+| staging           | 300000000003 |
+
+### AWS IAM user or Roles
 There are 2 options here
-1. Using AWS IAM AccessKey and Secret
-Option number 1 is useful for creation of user with AWS Access Key and secret. This is a common configuration. 
-2. Using AWS IAM Roles
-Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
+1. Set Up AWS IAM structure and use an AWS IAM User *spinnakerManaging* with AccessKey and Secret. This option is useful for creation of user with AWS Access Key and secret. This is a common configuration. 
+2. Using AWS IAM Roles. Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations. For Example the EC2 instance where Spinnaker is used to deploy AWS resources has the *ManagingRole* attached. The *ManagingRole* contains the AWS IAM policies required to manage AWS resources.
+
+### Create AWS IAM user
+- Access AWS IAM console
```

Comment:
@aleon1220 can you make this an ordered list instead of unordered?

Reply:
yes